perftests/scheduler_simulator/fake_worker_cli.go

```diff
@@ -0,0 +1,103 @@
+package scheduler_simulator
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/twitter/scoot/cloud/cluster"
+	"github.com/twitter/scoot/common/log/tags"
+	"github.com/twitter/scoot/runner"
+)
+
+/*
+FakeWorker sleeps for the amount of time defined in the command's argv[1] entry then
+returns exit code defined in argv[2]
+*/
+type FakeWorker struct {
+	node   cluster.Node
+	doneCh chan bool
+	state  runner.RunState
+	cmdId  string
+}
+
+func makeFakeWorker(n cluster.Node) *FakeWorker {
+	return &FakeWorker{
+		node:   n,
+		doneCh: make(chan bool),
+		state:  runner.PENDING,
+	}
+}
+
+func (fw *FakeWorker) Run(cmd *runner.Command) (runner.RunStatus, error) {
```

Comment:
Might want to at least vet the Argv here to avoid a panic if it's not well formed

Reply:
done